src/utils.js

```diff
@@ -227,7 +227,7 @@ export const getTimeOfDay = () => {
   }
 };
 
-export const hasPropChanged = (oldProps, newProps, key) => {
+export const hasPropChanged: boolean = (oldProps, newProps, key) => {
```

Comment:
Nit: this annotation shouldn't be necessary, Flow is generally pretty good about inferring this. You can still add it, if you think it's helpful, but it isn't required.